lib/flay.rb

```diff
@@ -63,7 +63,7 @@ def self.parse_options args = ARGV
 
     OptionParser.new do |opts|
       opts.banner  = "flay [options] files_or_dirs"
-      opts.version = Flay::VERSION
+      opts.version = VERSION
```

Comment:
If I build a FooFlay and it has a FooFlay::VERSION, then using `fooflay -v` should show fooflays version